hooks/useAppointment.tsx

```diff
@@ -188,9 +188,108 @@ export default function useAppointment(dispatchAppointment: React.Dispatch<IAppo
 		}
 	}
 
+	async function getAppointments(birthDate: string | null = null, nationalNumber: string | null = null) : Promise<object | undefined> {
+		Console.info("Getting appointments...");
+
+		let appointmentsURL = `
+			${Variables.DOMAINE_HTTP}/modules/Modules/QueueManagement/services/appointments.php?
+			id_project=${Variables.W_ID_PROJECT}
+			&serial=${Variables.SERIAL}
+		`;
+
+		if(birthDate) 	appointmentsURL += `&birth_date=${birthDate}`;
+		if(nationalNumber) 	appointmentsURL += `&registre_national=${nationalNumber}`;
+
+		try {
+			const response = await fetchRetry(appointmentsURL);
+			const data = await response.json();
+
+			if(!data) {
+				Console.error("Error when trying to get appointments: data is null", { fileName: "useAppointment", functionName: "getAppointments", lineNumber: 209, });
+				dispatchError({
+					type: ERROR_ACTION_TYPE.SETERROR,
+					payload: {
+						hasError: true,
+						errorCode: ERROR_CODE.B500,
+						message: "Unable to fetch ticket PDF (Data is null)",
+					},
+				});
+				return undefined;
+			}
+
+			if (data.status === 1) {
+				return data;
+			} else {
+				Console.error("Error when trying to get appointments: data status " + data.status ?? "undefined", { fileName: "useAppointment", functionName: "getAppointments", lineNumber: 209, });
```

Comment:
I'm a bit septical about those lineNumber, they seam to be pointing to the wrong line?